applications/GeoMechanicsApplication/custom_utilities/static_condensation_utility.h

```diff
@@ -28,16 +28,16 @@ namespace Kratos
 /**
  * @class GeoStaticCondensationUtility
  *
- * @brief This utilitiy condenses given degrees of freedom from any element stiffness matrix to model e.g. hinges
+ * @brief This utility condenses given degrees of freedom from any element stiffness matrix to model e.g. hinges
  *
  */
 
 class GeoStaticCondensationUtility
```

Comment:
It seems this was copied from structural mechanics, we might just be able to re-use that one.

Reply:
Yes, please :-)